File: ciam/src/main/java/com/crm/ciam/dto/BenutzerDTO.java
package com.crm.ciam.dto;

import java.time.LocalDateTime;
import java.util.List;

public record BenutzerDTO(
        Long id,
        String benutzername,
        String vorname,
        String nachname,
        String email,
        List<String> rollen,
        boolean aktiv,
        LocalDateTime createdAt,
        LocalDateTime updatedAt
) {}

File: ciam/src/main/kotlin/com/crm/ciam/dto/BenutzerDTO.kt
package com.crm.ciam.dto

import java.time.LocalDateTime

data class BenutzerDTO(
    val id: Long?,
    val benutzername: String,
    val vorname: String,
    val nachname: String,
    val email: String,
    val rollen: List<String>,
    val aktiv: Boolean,
    val createdAt: LocalDateTime?,
    val updatedAt: LocalDateTime?
)
